snapcraft/internal/meta.py

```diff
@@ -29,7 +29,10 @@
 
 from snapcraft import file_utils
 from snapcraft import shell_utils
-from snapcraft.internal import common
+from snapcraft.internal import (
+    common,
+    repo
+)
```

Comment:
not asking for a change, but refactors had taught me that these are easier to handle if they are each one liners :wink: